### ISSUE TITLE
Precompile fails

### ISSUE THREAD

**Issue by WaterQualityModels** [EnzymeAD/Enzyme.jl#145]
Trying to add Enzyme directly to new project after failure to add DiffEqFlux in different project. Adding Enzyme directly returns error:
```
(DiffEqFlux) pkg> add Enzyme
    Updating registry at `C:\Users\gregc\.julia\registries\General`
    Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Resolving package versions...
    Updating `C:\Users\gregc\OneDrive\Documents\Graduate Courses\Julia\DiffEqFlux\Project.toml`
  [7da242da] + Enzyme v0.7.0
    Updating `C:\Users\gregc\OneDrive\Documents\Graduate Courses\Julia\DiffEqFlux\Manifest.toml`
  [79e6a3ab] + Adapt v3.3.1
  [fa961155] + CEnum v0.4.1
  [7da242da] + Enzyme v0.7.0
  [e2ba6199] + ExprTools v0.1.6
  [61eb1bfa] + GPUCompiler v0.12.9
  [692b3bcd] + JLLWrappers v1.3.0
  [929cbde3] + LLVM v4.6.0
  [d8793406] + ObjectFile v0.3.7
  [21216c6a] + Preferences v1.2.2
  [189a3867] + Reexport v1.2.2
  [53d494c1] + StructIO v0.3.0
  [a759f4b9] + TimerOutputs v0.5.13
  [7cc45869] + Enzyme_jll v0.0.19+0
  [dad2f222] + LLVMExtra_jll v0.0.11+0
  [0dad84c5] + ArgTools
  [56f22d72] + Artifacts
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [f43a241f] + Downloads
  [b77e0a4c] + InteractiveUtils
  [b27032c2] + LibCURL
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [ca575930] + NetworkOptions
  [44cfe95a] + Pkg
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA
  [9e88b42a] + Serialization
  [6462fe0b] + Sockets
  [fa267f1f] + TOML
  [a4e569a6] + Tar
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode
  [deac9b47] + LibCURL_jll
  [29816b5a] + LibSSH2_jll
  [c8ffd9c3] + MbedTLS_jll
  [14a3606d] + MozillaCACerts_jll
  [83775a58] + Zlib_jll
  [8e850ede] + nghttp2_jll
  [3f19e933] + p7zip_jll
Precompiling project...
  Progress [========================================>]  1/1
  ✗ Enzyme
0 dependencies successfully precompiled in 3 seconds (16 already precompiled)
1 dependency errored
```

**Comment by vchuravy** [comment]
Was does `using Enzyme` yield?

**Comment by vchuravy** [comment]
Hm, very odd. We do have CI for Windows https://github.com/wsmoses/Enzyme.jl/runs/3768839737 and I haven't seen a failure like this before.

What version of Windows are you using? What is your `versioninfo()
`

**Comment by WaterQualityModels** [comment]
Okay - thanks for the quick feedback. I'll wipe everything and try with new installs from the top.

Windows 10 Home, Version 2004, OS Build 19041.1237, Windows Feature Experience Pack 120.2212.3530.0
```
julia> versioninfo()
Julia Version 1.6.0
Commit f9720dc2eb (2021-03-24 12:55 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)
Environment:
  JULIA_CMDSTAN_HOME = C:\Users\gregc\.cmdstan
  JULIA_EDITOR = code
  JULIA_NUM_THREADS = 4
```

**Comment by WaterQualityModels** [comment]
I uninstalled VSCode and Julia and installed each new (upgrading to 1.6.3). Same error resulted. 

Local solution: The artifacts folder referenced in the error was completely empty in ~/.julia, so I moved ~/.julia to a different tmp folder, reinstalled 1.6.3 again and then ran `]up` in a new project to re-fetch everything from the registry fresh. Enzyme successfully precompiles now. DiffEqFlux successfully compiles in turn. 

Not sure if this is of interest to you all to look at further, so I'll leave it open for you to close. Thanks again for the feedback.
